test_utils/load_tests.py

```diff
@@ -1,17 +1,5 @@
-import os
 import sys
 
-APPS_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
-BASE_DIR = os.path.join(APPS_DIR, "django_socio_grpc")
-sys.path.append(APPS_DIR)
```

Comment:
why deleting all of that ? I understand that conftest is launch automatically with the test but I used it sometime without test command like a real project with fakeApp project for some tests.

Reply:
working this way as it only affect load_tests